docker-compose.yml

```
@@ -34,11 +34,19 @@ services:
 
   db:
     image: postgres:9.5
+    ports:
+      - 5432:5432
```

Can we change the host side port to something else? The reason behind it is that it might create a conflict for given developer who would want to start the project while also running some database on said port.
```suggestion
      - 6866:5432
```

---

Sounds good